.github/workflows/test.yml

```diff
@@ -123,14 +131,9 @@ jobs:
 
       - run: cabal update
 
-      # Need this to work around filepath length limits in Windows
-      - name: Shorten binary names
+      - name: "HLint via ./fmt.sh"
```

Comment:
why moving hlint here? just curious

Reply:
1. To have the same sequence of actions between workflows.

2. To do bootstrap first & then do everything else. Bootstrap steps are ~ instant anyway, except for fast `cabal v2-update`. So there is no need to not look/track what is happening in/during the `hlint` oneliner script, does it install something additional & does that depend on the Hackage state. & install can be made faster if what it wants to install is already in the cache. So there goes the quick bootstrapping phase & then quick `hlint` install & goes code that analyzes & tests the project.

Overall 1 run of `hlint` is mostly enough. It can run just on the latest GHC. & so if it needs to look between CPP - in time - all the code would get looked through. Would ping upstream on that action, there was almost a month after submission of the action.